frontend/src/page/Signup.jsx

```diff
@@ -1,10 +1,11 @@
 import React, { useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 import { signInWithPopup } from 'firebase/auth';
-import './signup.css';
+import './loginsignup.css';
 import googlelogo from '../assets/google_logo.svg';
 import { auth, provider } from '../services/firebase';
-import createUser from '../api/user';
+import { createUser } from '../api/user';
+import Form from '../components/Form';
 
 function Signup() {
   const [userName, setUserName] = useState('');
```

Comment:
The same changes for Login regarding variable naming needs to be adjusted here as well in Signup. Focus on consistency and good practices.